pkg/config/config.go

```diff
@@ -29,9 +29,14 @@ type Config struct {
 }
 
 type IgnoreEntry struct {
-	ID          string    `toml:"id"`
-	IgnoreUntil time.Time `toml:"ignoreUntil"`
-	Reason      string    `toml:"reason"`
+	ID             string    `toml:"id"`
+	IgnoreUntil    time.Time `toml:"ignoreUntil"`
+	Reason         string    `toml:"reason"`
+	IncludeAliases *bool     `toml:"includeAliases"` // Use pointer to tell apart unset (treated as true) and false
```

Comment:
If we default this to true, we should just start with not including this flag at all. It's easier to add a flag later than to remove it. This will likely also simplify the implementation a lot. 

It's hard to come up with a reasonable and unconfusing use case where users will want to set this to false. 

@another-rex WDYT?

Reply:
Removing it for now sounds good to me.

Reply:
Happy to remove this.